cedar-policy/src/frontend/utils.rs

```diff
@@ -102,6 +103,47 @@ impl PolicySpecification {
     }
 }
 
+/// Represents a schema in either schema format
+#[derive(Debug, Serialize, Deserialize)]
+pub enum Schema {
+    /// Schema in the Cedar schema format. See <https://docs.cedarpolicy.com/schema/human-readable-schema.html>
+    #[serde(rename = "human")]
+    Human(String),
+    /// Schema in Cedar's JSON schema format. See <https://docs.cedarpolicy.com/schema/json-schema.html>
+    #[serde(rename = "json")]
+    Json(JsonValueWithNoDuplicateKeys),
+}
+
+impl Schema {
+    pub(super) fn parse(
+        self,
+    ) -> Result<(crate::Schema, Box<dyn Iterator<Item = SchemaWarning>>), String> {
```

Comment:
Nit: it would feel more consistent to have both `PolicySet::parse` and `Schema::parse`, or both `PolicySet::try_into` and `Schema::try_into`

Reply:
Renamed to parse()